### PR TITLE
cmd/bnscli: Fix boolean flag bug

### PR DESCRIPTION
**Pull request by orkunkl**
Changed boolean values to string as input. 

Related issue:
https://stackoverflow.com/a/27411724/12566604

### CODE REVIEW

File: cmd/bnscli/clitests/account_register_domain.test [hunk 1]
```diff
@@ -5,7 +5,7 @@ set -e
 bnscli register-domain \
 		-domain "mydomain" \
 		-admin "92066456B2BE7F1934624087D98C203A87F7752C" \
-		-superuser \
+		-superuser "false" \
```
Comment (davepuchyr) [inline cmd/bnscli/clitests/account_register_domain.test]: Seems to me that this should have been "true" and the following superuser should be "false".

File: cmd/bnscli/clitests/account_register_domain.test [hunk 2]
```diff
@@ -14,10 +14,10 @@ echo
...
 
 bnscli register-domain \
-		-domain "another-domain" \
-		-admin "92066456B2BE7F1934624087D98C203A87F7752C" \
-		-superuser false \
-		-account-renew "1h 4m" \
-	| bnscli with-account-msg-fee -path 'account/first_msg' -amount "5 IOV" \
-	| bnscli with-account-msg-fee -path 'account/second_msg' -amount "9 IOV" \
-	| bnscli view
+                 		-domain "another-domain" \
```
Comment (davepuchyr) [inline cmd/bnscli/clitests/account_register_domain.test]: What's going on with the indentation?

File: cmd/bnscli/clitests/account_register_domain.test.gold [hunk 3]
```diff
@@ -39,7 +38,7 @@
 					}
 				}
 			],
-			"account_renew": 2592000
+			"account_renew": 3840
```
Comment (davepuchyr) [inline cmd/bnscli/clitests/account_register_domain.test.gold]: Why did this change?

File: cmd/bnscli/cmd_query.go [hunk 4]
```diff
@@ -41,7 +41,7 @@ Execute a ABCI query and print JSON encoded result.
 			"Tendermint node address. Use proper NETWORK name. You can use BNSCLI_TM_ADDR environment variable to set it.")
 		pathFl        = fl.String("path", "", "Path to be queried. Must be one of the supported.")
 		dataFl        = fl.String("data", "", "individual query data. Format depends on the queried entity. Use 'id/version' for electoraterules, electorates")
-		prefixQueryFl = fl.Bool("prefix", false, "If true, use prefix queries instead of the exact match with provided data.")
+		prefixQueryFl = fl.String("prefix", "false", "If true, use prefix queries instead of the exact match with provided data. [true/false]")
```
Comment (davepuchyr) [inline cmd/bnscli/cmd_query.go]: Nice

File: cmd/bnscli/clitests/account_register_domain.test.gold (hunk 3 above)
Comment (orkunkl) [reply cmd/bnscli/clitests/account_register_domain.test.gold]: The test were not working properly apparently. 12345 h = 44442000 and 1h4m = 3840

File: cmd/bnscli/clitests/account_register_domain.test (hunk 2 above)
Comment (orkunkl) [reply cmd/bnscli/clitests/account_register_domain.test]: Very weird. Fixed it

**Pull request merged**